### PR TITLE
Autoscoring and detail view bugbash changes

### PR DESCRIPTION
**Pull request by mehalshah**
- [ ] Hide the submission section in the detail view (will go away with the redesign)
- [ ] Display whether the application meets scholarship criteria
- [ ] Update links for the rubric
- [ ] Pass the year of the application to the detail view, used when displaying the list of workshops
- [ ] Change spacing for submission section
- [ ] When returning percents for the principal approval, add a % symbol
- [ ] Compute total course hours when applications are created (Note: We have to do this in the prod console after this ships)
- [ ] Update abilities in ability.rb to allow users to mark applications as not needing principal approval
- [ ] Autoscoring with the new standards

### CODE REVIEW

File: dashboard/app/models/pd/application/teacher1920_application.rb [hunk 1]
```diff
@@ -391,5 +391,107 @@ def self.filtered_labels(course)
...
+        NO
+      else
+        'Reviewing incomplete'
+      end
+    end
+
+    def meets_scholarship_criteria
+      responses = response_scores_hash.slice(*SCHOLARSHIP_QUESTIONS).values
+
+      # Edge case for plan to teach
```
Comment (clareconstantine) [inline dashboard/app/models/pd/application/teacher1920_application.rb:486]: does this comment mean you have yet to handle this edge case?

File: apps/src/code-studio/pd/application_dashboard/detail_view_application_specific_questions.jsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ export default class DetailViewApplicationSpecificQuestions extends React.Compon
       principalApproval: this.props.initialPrincipalApproval
     };
 
-    this.sectionHeaders = props.applicationType === TEACHER ? _.omit(TeacherSectionHeaders, ['section5Submission']) : FacilitatorSectionHeaders;
+    this.sectionHeaders = props.applicationType === TEACHER ? _.omit(TeacherSectionHeaders, ['section5Submission', 'section6Submission']) : FacilitatorSectionHeaders;
```
Comment (clareconstantine) [inline apps/src/code-studio/pd/application_dashboard/detail_view_application_specific_questions.jsx:49]: I don't think we need section5Submission in this list anymore - it doesn't exist anywhere else (I think they were renumbered)

**Review by clareconstantine — APPROVED** [review]

File: dashboard/app/models/pd/application/teacher1920_application.rb (hunk 1 above)
Comment (mehalshah) [reply dashboard/app/models/pd/application/teacher1920_application.rb:486]: Correct - but upcoming changes negate this

File: apps/src/code-studio/pd/application_dashboard/detail_view_application_specific_questions.jsx (hunk 2 above)
Comment (mehalshah) [reply apps/src/code-studio/pd/application_dashboard/detail_view_application_specific_questions.jsx:49]: We do need this for rendering old applications

Comment (clareconstantine) [reply apps/src/code-studio/pd/application_dashboard/detail_view_application_specific_questions.jsx:49]: ahh, gotcha.

**Pull request merged**